### ISSUE TITLE
Define table-handle gets parsed as an error

### ISSUE THREAD

**Issue by gustason** [BalticAmadeus/AblFormatter#245]
For example,
``` abl
define INPUT-OUTPUT PARAMETER TABLE-HANDLE hNode.   
define OUTPUT PARAMETER TABLE-HANDLE hData.   
define OUTPUT PARAMETER TABLE-HANDLE hSchema. 
```
all get marked as errors:
![image](https://github.com/user-attachments/assets/ae7489ca-cca8-42bf-abe5-2a25bcde3ad3)

Likewise, define dataset-handle gets marked as an error as well:
``` abl
define INPUT  PARAMETER DATASET-HANDLE hDset.
define OUTPUT PARAMETER TABLE-HANDLE ttHandle.
```

**Comment by eglekaz** [comment]
Registered: https://github.com/usagi-coffee/tree-sitter-abl/issues/98

**Comment by eglekaz** [comment]
Fixed with version 0.0.44 of tree-sitter-abl
